setup.py

```diff
@@ -26,5 +26,6 @@
         'djangorestframework>=3.2.0,<3.4.0',
         'djangorestframework-jwt>=1.6.0,<=1.7.2',
         'edx-opaque-keys>=0.1.2,<1.0.0',
+        'edx-djangorestframework-oauth==1.0.1',
```

Comment:
@clintonb worth mentioning: if we get rid of the `edx-djangorestframework-oauth` fork as proposed in https://github.com/edx/edx-platform/pull/11559, we'll need to carefully order imports here, too, so that `django-oauth2-provider` is shadowed.